takwimu/sql/age_group_cervical_cancer.sql

```diff
@@ -0,0 +1,62 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 10.6
```

Comment:
Is it possible to dump data via docker so that we have the exact/consistent production PG version (9.5 Ithink)?